landoui/static/src/js/pages/RevisionPage.js

```diff
@@ -0,0 +1,33 @@
+/* Simple class to manage events and state on the /revision/{id} page */
+Lando.RevisionPage = class {
```

Comment:
Do you mean we should create 1 plugin for the entire 1 "RevisionPage" or a plugin for the collapsible that I made?